themes/hugo-rladies/layouts/directory/single.html

```diff
@@ -1,163 +1,135 @@
 
 {{ define "main" }}
-<!-- Selects -->
-{{ $directory := ( .Scratch.Get "directory" ) }}
-{{ $ints := slice }}
-{{ $langs := slice }}
-{{ $loc := slice }}
-{{ $countries := slice }}
-
-{{ range $directory }}
-  {{ with .interests }}
-      {{ $ints = $ints | append  . }}
-    {{ end }}
-    
-    {{ with .languages }}
-      {{ $langs = $langs | append  . }}
-    {{ end }}
-    
-    {{ with .location }}
-      {{ $tmp := newScratch }}
-      {{ with .country }}
-        {{ $tmp.Set "loc" . }}
-        {{ $countries = $countries | append  . }}
-      {{ end }}
-      
-      {{ with .state }}
-        {{ $tmp.Add "loc" ( . | printf ", %s") }}
-      {{ end }}
-      
-      {{ with .city }}
-        {{ $tmp.Add "loc" ( . | printf ", %s") }}
-      {{ end }}
-      
-      {{ $loc = $loc | append  ($tmp.Get "loc") }}
-  {{ end }}
-{{ end }}
-{{ $ints := $ints | uniq | sort }}
-{{ $langs := $langs | uniq | sort }}
-{{ $loc := $loc | uniq | sort }}
-{{ $countries := $countries | uniq | sort }}
-<!-- -->
-
-  <div class="entry-content">
-		{{ .Content }}
-	</div>
-  <div id="counters" class="row justify-content-center section topspace">
-  	<div class="col-sm-5 col-md-4 text-center fadeInUp" data-wow-duration="500ms">
-    	<div class="block">
-    		<div class="icon-box">
-    			<h3>{{ i18n "counter_entries" | markdownify }}</h3>
-    			<i class="fa-solid fa-user"></i>
-    			<p class="count" data-count="{{ $directory | len }}">0</p>
-    		</div>
-  		</div>
-  	</div>
-  </div>
-  <div class="flex-container" id="directory-content">
-    <div id="selection">
-      
-      <div class="input-group w-100">
-        <div class="input-group-prepend" for="filter-interests">
-          <label class="input-group-text" style="min-width:120px;" >Interests</label>
-        </div>
-        <select class="custom-select" id="filter-interests" multiple data-group="interests">
-          {{ range $ints }}
-            <option value="{{. | urlize }}" >{{.}}</option>
+<div class="container my-5">
+  <div class="row g-4">
+
+    <div class="col-lg-4">
+      <div class="card shadow-sm mb-4 text-center">
+        <div class="card-body">
+          {{ if .Params.image }}
+              <img src="{{ .Params.image }}" class="img-fluid mb-3 border border-3 border-light shadow-sm" alt="{{ .Params.name }}" style="object-fit: cover;">
+              {{ with .Params.photo.credit }}
+                <p class="card-text text-muted small">Photo credit: {{ . }}</p>
```

Comment:
on each directory page it'd be nice to have a sentence in small reminding "Is this your entry? Update it via blablabla".

Reply:
this is a good idea, should be easy enough to implement.

Also, that interest you point to is not something that appears on the website. Its one I made in the mock directory to pick up if the js broke because of weird chars and symbols in the interest field. 
We are working on making a curated list of interest (in the new directory airtable form you must choose from a set), but I made this to be compatible with old entries that still may have weird characters in there.